src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -140,7 +146,8 @@ private void configMotors() {
         turretMotor.configReverseSoftLimitEnable(true);
         turretMotor.configSupplyCurrentLimit(turretCurrentLimit);
         turretMotor.setNeutralMode(NeutralMode.Brake);
-        turretMotor.configClosedLoopPeakOutput(TURRET_SLOT_ID, 50);
+        turretMotor.configClosedLoopPeakOutput(TURRET_SLOT_ID, 0.1);
```

Comment:
We should consider making this a constant with a good name to help prevent value from being too high again.